parachains/integration-tests/emulated/common/src/lib.rs

```diff
@@ -27,6 +27,7 @@ decl_test_relay_chains! {
 			MessageQueue: polkadot_runtime::MessageQueue,
 			XcmConfig: polkadot_runtime::xcm_config::XcmConfig,
 			SovereignAccountOf: polkadot_runtime::xcm_config::SovereignAccountOf,
+			AllPallets: polkadot_runtime::AllPalletsWithSystem,
 			System: polkadot_runtime::System,
 			Balances: polkadot_runtime::Balances,
```

Comment:
not sure if we even need to list the pallets, and pallets_extra. 
may be optionally, if there is some use case for it.